plugins/modules/nmcli.py

```diff
@@ -80,6 +82,13 @@
         type: str
         choices: [ datagram, connected ]
         version_added: 5.8.0
+    slave_type:
+        description:
+            - Type of the device of this slave's master connection (eg, "bond").
+            - If I(master) defined this option is mandatory.
```

Comment:
A little wordsmithing - just a suggestion, feel free to refute it.
```suggestion
            - Mandatory if I(master) is defined.
```

Reply:
I fail to see the code that enforces that dependency. It should either test this explicitly somewhere, or it should have a `required_by` in the module definition

Reply:
@russoz Hi, thanks for review.
Briefly and clearly means good)

`required_by` for `slave_type` added to module argspecs.

Reply:
Hi @heaveaxy Thanks for the adjustments. I failed to find the `required_by`, can you please double check if it was pushed to the PR? And keep in mind that if you use that `required_by`, it will apply to **any** action that uses the parameter `master`. Please make sure that by using it for this case we are not disrupting another. TIA

Reply:
Actually (I have checked it) if one of `master` or `slave-type` is defined, another must be defined too. It may be described in argspec by `required_together`, BUT...
But we have types as `bond-slave` that works as alias for `slave-type` = `bond`, where `slave-type` property may not be defined, but `master` must.
I implemented this checks another way by creating method for additional validation.